### DISCUSSION TITLE
Welcome to liteshell `lsh` project discussions!

### DISCUSSION THREAD

**Discussion by thenoblet** [codemindshub/liteshell#1] [Announcements]
## 👋 Welcome to CodeMinds first project together `liteshell` or `lsh` for short!
  We’re using Discussions as a place to connect with other members of the CodeMinds community on this project. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  📢 **Be patient with others and be considerate. We are here to learn and collaborate.**

**Comment by OmondiHillaryA1997** [comment]
Am Hillary. Excited to be here and looking forward not only for this collaboration but more. Aiming to bring out, the best of me through the help of this able team of Mentors.

Hope some day, we will look back and appreciate the Process.
Grateful to be here.
